File: testdata/fixtures/csharp/ThemePreview.cs
using System;

namespace Kat.Rendering;

public sealed class ThemePreview
{
    private const string DefaultTheme = "Dracula";

    public ThemePreview(string name)
    {
        Name = string.IsNullOrWhiteSpace(name) ? DefaultTheme : name;
    }

    public string Name { get; }

    [Obsolete("Use Render instead.")]
    public string Render()
    {
        var title = $"{Name} Preview";
        return $"{title}: enabled={true}";
    }
}

File: testdata/fixtures/java/ThemePreview.java
package kat.preview;

public final class ThemePreview {
  private static final String DEFAULT_THEME = "Dracula";

  public static void main(String[] args) {
    System.out.println(render(DEFAULT_THEME));
  }

  static String render(String name) {
    return "theme:" + name;
  }
}
